torch/_inductor/codegen/cpp_prefix.h

```diff
@@ -77,22 +92,33 @@ Welford<T> welford_combine(const Welford<T> &a, const Welford<T> &b) {
   auto result = Welford<T>{
     a.mean + delta * wb_over_w,
     a.m2 + b.m2 + delta * delta * a.weight * wb_over_w,
-    new_weight
+    new_weight,
+    a.index + b.index,
```

Comment:
Does the index mean we can change the above bail-out for `weight == 0` to `index == 0` and cover the vectorized cases as well. That would allow the blendv to be removed.

Reply:
We can't allow the blendv to be removed as `weight_recps` may not include `b.weight / new_weight`.

Reply:
Understood what you meant. Removed the `blendv `.